tools/opt/opt-passes.h

```diff
@@ -0,0 +1,21 @@
+#ifndef SPIRV_TOOLS_OPT_PASSES_H_
+#define SPIRV_TOOLS_OPT_PASSES_H_
+
+#include <vector>
+#include <string>
+
+namespace spvtools {
+
+// Performance optimization flags expanded by -O.
+//
+// Using the -O flag in spirv-opt's command line is equivalent to specifying
+// the following flags in this exact sequence.
+std::vector<std::string> OptimizationPasses = {
+    "--inline-entry-points-opaque",
```

Comment:
See my recommendations for -Os. I would include those here as well. They are designed to eliminate all loads and stores to function scope variable which is a performance win besides a size win.

I would not use opaque inlining here. It was not designed with performance in mind (and it doesn't even quite solve the problem it was originally designed for). I am currently writing a "default" inliner which I think might be more acceptable here, but until that is ready I think the exhaustive inliner is going to be our best bet. Without some form of inlining, the other size-reduction passes will be much less effective for shaders with function calls.

Reply:
Thanks.  As a starting point, I've made both lists identical for now.  I've added comments stating that they're in flux.